index.html

```diff
@@ -1,161 +1,174 @@
-        </li>
-      </ul>
-    </section>
-
-   <section class="search">
-      <h3 class="search-title">Заинтересовались?</h3>
-      <p class="search-description">Укажите предполагаемые даты поездки,и мы покажем вам лучшие предложения гостиниц
-        в Седоне</p>
-      <a class="search-button" href="#">Поиск гостиницы в Седоне</a>
-    </section>
-
-    <section class="newsletter">
-      <h2 class="page-footer-title">Подпишитесь на рассылку</h2>
-      <p class="advantages-description">Только полезная информация и никакого спама, честное бойскаутское!</p>
-      <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
-        <label for="newsletter-email">Email</label>
-        <input class="field" placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email" required>
-        <button class="button newsletter-button" type="submit">Подписаться</button>
-      </form>
-    </section>
-
-  </main>
-
-  <footer>
-    <section class="footer social">
-      <h2 class="page-footer-title">Контакты</h2>
-      <ul class="footer-social-list">
-        <li class="footer-social-item">
-          <a class="button" href="#">
-            <span class="visually-hidden">ВКонтакте</span>
-          </a>
-        </li>
-        <li class="footer-social-item">
-          <a class="button" href="#">
-            <span class="visually-hidden">Твиттер</span>
-          </a>
-        </li>
-
-
-        <li class="footer-social-item">
-          <a class="button" href="#"></a>
-          <span class="visually-hidden">Youtube</span>
-          </li>
-        </ul>
-      </section>
-
-    <a class="footer-contacts-address-phone" href="tel:78128121212">+7 (812) 812-12-12</a>
-    <p class="about-text"><a class="htmlacademy-link" href="https://htmlacademy.ru/"></a>.html academy</p>
-  </footer>
-
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+
+<body>
+  <header class="page-header">
+    <a class="navigation-logo" href="#">
+      <img class="page-header-logo" src="logo.svg" width="140" height="70" alt="Логотип города Седона.">
+    </a>
+
+
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation-item">
+          <a class="navigation-link" href="index.html">Главная</a>
+        </li>
+
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">О Седоне</a>
+        </li>
+
+        <li class="navigation-item">
+          <a class="navigation-link" href="catalog.html">Гостиницы</a>
+        </li>
+      </ul>
+
+      <ul class="navigation-user">
+        <li class="navigation-user-item">
+          <a class="navigation-user-link navigation-user-link-search" href="#">
+            <span class="visually-hidden">Поиск</span>
+      <img class="search-button" src="search.svg" width="44" height="64">
+
+
+          </a>
```

Comment:
Эта иконка - декоративное изображение, вставлять в разметку тегом `img` не нужно. Должно быть так:
```suggestion
          <a class="navigation-user-link navigation-user-link-search" href="#">
            <span class="visually-hidden">Поиск</span>


          </a>
```